WF_Proton_Launcher/Main.cpp

```diff
@@ -105,6 +105,12 @@ std::string registry_option;
 
 std::string fullscreen_option = " -fullscreen:0 ";
 
+std::string language = "-language:en";
+
+std::string lang;
+
+std::string supported_languages[13] = {"de", "es", "fr", "it", "ja", "ko", "pl", "pt", "ru", "tc", "uk", "zh", "en"};
```

Comment:
If this is required then I'd prefer if this was `const` and all uppercase, preferrably also relocated up top to the rest of the internal constants.